core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message.properties

```diff
@@ -0,0 +1,23 @@
+#The MIT License
+#
+#Copyright (c) 2021, CloudBees, Inc.
+#
+#Permission is hereby granted, free of charge, to any person obtaining a copy
+#of this software and associated documentation files (the "Software"), to deal
+#in the Software without restriction, including without limitation the rights
+#to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#copies of the Software, and to permit persons to whom the Software is
+#furnished to do so, subject to the following conditions:
+#
+#The above copyright notice and this permission notice shall be included in
+#all copies or substantial portions of the Software.
+#
+#THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#THE SOFTWARE.
+
+ExecutorsWarning=You should set up distributed builds. Building on the controller node can be a security issue. See <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">the documentation</a>.
```

Comment:
As per https://issues.jenkins.io/browse/JENKINS-65291 can we say something like this? "You should disable builds on the master node. You can do this be setting the number of executors to zero."

Also rename the warning to "ControllerExecutorsWarning" or something like this.